src/heap_utils.cc

```diff
@@ -448,30 +448,60 @@ void CreateHeapSnapshotStream(const FunctionCallbackInfo<Value>& args) {
 void TriggerHeapSnapshot(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
   Isolate* isolate = args.GetIsolate();
-  CHECK_EQ(args.Length(), 2);
+  CHECK_EQ(args.Length(), 3);
   Local<Value> filename_v = args[0];
   auto options = GetHeapSnapshotOptions(args[1]);
+  Local<String> path_v = args[2].As<String>();
```

Comment:
`getValidatedPath` can return a Uint8Array, so this is an unsafe cast.

Reply:
Fixed!